src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -2509,7 +2509,7 @@ End Class
             Await VerifyItemExistsAsync(markup, "a")
             Await VerifyItemExistsAsync(markup, "b")
             Await VerifyItemExistsAsync(markup, "c")
-            Await VerifyItemIsAbsentAsync(markup, "Equals")
+            Await VerifyItemExistsAsync(markup, "Equals")
```

Comment:
This is an override of the instance method `Object.Equals`. Should we be explicitly filtering it out?